### ISSUE TITLE
Add ReadMe

### ISSUE THREAD

**Issue by vjjan91** [BirdEcoEvoLab/BirdLab-Protocols#1]
It would be useful to add a detailed readme that links to all the google docs that @vvrobin has created!

**Comment by vjjan91** [comment]
@stachyris tagging you here.

**Comment by stachyris** [comment]
Got it @vjjan91!! Will work on it soon

**Comment by vvrobin** [comment]
Hi Vinay and Vijay
Need access on this doc set - it asks me to create a fork if I want to edit

Also, I wonder if some of the Lab Manual items we can keep internal. For example on the hiring, sexual harassment issues etc. I am not sure if making that public opens it up to institutional and public scrutiny. Some of these are suggestions/ideas in good-faith.

But lab protocols etc are not really any secret - that’s what everyone does and uses.

What do you think?

Robin

**Comment by vjjan91** [comment]
I agree, Robin! Please let us(ie @stachyris or the main repository maintainers) know about what documents you would want uploaded/added as a link/directly included in the main page README.

@stachyris can you provide access/edit privileges if @vvrobin does not have it already?

**Comment by stachyris** [comment]
Agreed, @vjjan91. 
@vvrobin has access. Earlier there was some issue with his login, hence it wasn't showing up.

-Vinay
